README.md

```diff
@@ -57,6 +57,13 @@ the [API documentation](docs/cluster_spec.md).
 5. Install the [foundationDB client package](https://www.foundationdb.org/download).
 6. If you want to modify the manifests you currently need [yq](https://github.com/mikefarah/yq), we use the `v4.6.1` version.
 
+### Dockerfile usage
+
+Instead of installing the dependencies to your local machine, you can use `Dockerfile.ci` to create a development environment. If Docker is installed, these steps suffice:
+
+1. docker build -f Dockerfile.ci -t fdb-kubernetes-operator-dev
+2. docker run -it -v <path to fdb-kubernetes-operator>:/fdb-kubernetes-operator fdb-kubernetes-operator-dev /bin/bash
```

Comment:
I suspect that kind won't work in that setting but kind is only used in the ci so that should be fine and we can fix that in the next step.

Reply:
I think we should including running make commands in this section of the docs, and add caveats about the commands that won't be supported.